pkg/configuration/configuration.go

```diff
@@ -291,3 +298,8 @@ func (c *Config) GetAllRegistrationServiceParameters() map[string]string {
 	}
 	return vars
 }
+
+// GetDeactivationDomainsExcludedList returns a string of space-separated domains that should be excluded from automatic user deactivation
```

Comment:
Also, maybe it's worth clarifying that we are talking about **user email** domains here?

Reply:
Done